### PR TITLE
Issue #5465: Multiple versions of cron_queue_test.module and .info in Backdrop core

### LINKED ISSUE #5465: Fixed: Remove duplicate instances of cron_queue_test.module and cron_queue_test.info files.
## Description of the bug

There are two different versions of file cron_queue_test.module in Backdrop Core. Since only one of them can be loaded, the other is superfluous. They are at:

* core/modules/system/tests/cron_queue_test/cron_queue_test.module
* core/modules/system/tests/cron_queue_test.module

Similarly, there's

* core/modules/system/tests/cron_queue_test/cron_queue_test.info
* core/modules/system/tests/cron_queue_test.info

The second of each pair is what actually gets loaded.

## Steps To Reproduce

Examine the directory structure. Or check the PR sandbox for https://github.com/backdrop/backdrop/pull/3910.

## Appropriate fix

Delete the superfluous files (or rather, move the the two files from `tests` directory into `tests/cron_queue_test` directory and replace the two that are there).

### PR DESCRIPTION
**Pull request by bugfolder**
Fixes https://github.com/backdrop/backdrop-issues/issues/5465.

### CODE REVIEW

File: core/modules/system/tests/cron_queue_test/cron_queue_test.info [hunk 1]
```diff
@@ -1,7 +1,7 @@
-name = "Cron Queue test"
-description = "Support module for the cron queue runner."
-backdrop = 1.x
-package = Testing
-version = BACKDROP_VERSION
 type = module
+name = Cron Queue test
```
Comment (klonos) [inline core/modules/system/tests/cron_queue_test/cron_queue_test.info:1]: Trivial, I know, but I preferred it when type was listed last.

**Review by klonos — APPROVED** [review]
LGTM 👍🏼

File: core/modules/system/tests/cron_queue_test/cron_queue_test.info (hunk 1 above)
Comment (bugfolder) [reply core/modules/system/tests/cron_queue_test/cron_queue_test.info:1]: Fine by me. That's just how it was in the (not quite duplicate) file that was being loaded. Now fixed.

**Comment by quicksketch** [comment]
Merged into 1.x and 1.21.x. Thanks!

**Pull request merged**